src/content/docs/ts-sdk/outputs/rtp.mdx

```diff
@@ -313,6 +316,33 @@ Audio output encoder preset.
     - `voip` - Recommended for VoIP and videoconferencing applications, prioritizing listening quality and speech intelligibility.
     - `lowest_latency` - Recommended **only** when achieving the lowest possible latency is the highest priority.
 
+--- 
+
+#### sampleRate
+Sample rate.
+
+- **Type**: `u32`
+- **Default value**: `48000`
+- **Supported values**: `8000`, `16000`, `24000`, `48000`
+
+---
+
+#### forwardErrorCorrection
+Specifies whether the stream uses forward error correction. It’s specific for the `Opus` codec. For more information, visit RFC specification sections [2.1.7](https://datatracker.ietf.org/doc/html/rfc6716#section-2.1.7)
```

Comment:
"Specifies" makes sense on input where we do not control that and are providing information to the smelter. Here we are configuring, controling or enabling error correction.
e.g.
Configure whether the stream should include redundant data that can be used for error correction in case of packet loss.